### ISSUE TITLE
Hypothesis & Data

### ISSUE THREAD

**Issue by paleomitchelljs** [Kas0141/Tasks#1]
Your posted hypothesis is that East Coast blue jay's live longer than West Coast blue jays but you did not post any data that contain lifespan?

Also, you've got a second, different hypothesis that decreased population size is related to lifespan, and there's no data on that here, either.

**Comment by Kas0141** [comment]
The Data for the hypothesis is in Project under ebd-sample. Sorry about that.

**Comment by Kas0141** [comment]
Hello,

The data for my hypothesis is in Project under ebd-sample.

Kaylea Stover

**Comment by paleomitchelljs** [comment]
Yes, I saw your data. There is no column for "lifespan" nor any way I can see to calculate lifespan of an individual bird from these data.

**Comment by Kas0141** [comment]
There is a column for the ages when they were sighted and I was going to use that if that would work, because it looks like they can range from 5-35.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Kas0141** [comment]
If you don’t think that is a good idea I can do something else or reword my hypothesis. I appreciate your help.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by paleomitchelljs** [comment]
There is not a column for ages when they were sighted that ranges from 5 - 35. There is a column for AGE/SEX which is almost entirely blank, except for a few occasions where someone put "age unknown".

This is eBird data. It's literally just a record of where/when some person saw a bird. You can't tell a bird's age from looking at it, just whether it's a juvenile/female or adult male (hence the combined age/sex column). 

Make sure you fully understand the limits of any data you end up choosing.

**Comment by Kas0141** [comment]
Oh okay, would testing a hypothesis like, “There are more Blue Jay’s on the East Coast than on the West Coast because there is a larger population of Stellar Jay’s on the West Coast” work? To test if there is competition between them and where there are more Stellar Jay’s there will be less Blue Jay’s.

Thanks,
Kaylea Stover

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by paleomitchelljs** [comment]
(1) You can't really say "there are more". You can only say "they are seen more often". Think about what your data are and how they were generated. You'll get more people recording seeing blue jays in places where there are more people, even if the number of birds is the same, for instance (and other ways).

(2) Even if you could determine exactly how many blue jays were on East vs West coasts....How would you know any difference you saw was *because* of Stellar's jays? Are Stellar's jays the only difference between East and West Coast that could possibly explain a difference in blue jay abundance?
